sdk/core/azure-core/inc/azure/core/uuid.hpp

```diff
@@ -8,32 +8,40 @@
 
 #pragma once
 
-#include "azure/core/platform.hpp"
-
 #include <array>
-#include <cstdint> // defines std::uint8_t
-#include <cstring>
-#include <stdint.h> // deprecated, defines uint8_t in global namespace. TODO: Remove in the future when references to uint8_t and friends are removed.
+#include <cstdint>
 #include <string>
 
 namespace Azure { namespace Core {
   /**
    * @brief Universally unique identifier.
    */
   class Uuid final {
+  public:
+    /**
+     * @brief Represents a byte array where the UUID value can be stored.
+     *
+     */
+    using ValueArray = std::array<std::uint8_t, 16>;
 
   private:
-    static constexpr size_t UuidSize = 16;
```

Comment:
Why did we remove this named constant?

Reply:
It is not used anywhere anymore. In place of it, where it is needed, we use `m_uuid.size()`, which is a constexpr, so effectively the same constant.

Reply:
I see `16` used now, as an unnamed constant.

Reply:
Yes, but that should be the only instance of `16` being used. The least should be replaced with `m_uuid.size()` which is effectively a constexpr 16.